src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -34,19 +34,18 @@ public class AddCommand extends Command {
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_REMARK + "REMARK "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_ROOM + "1 "
             + PREFIX_BOOKING_PERIOD + "2023-01-01 08:00 to 2023-01-02 12:00 "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_REMARK + "Requested extra Queen's size bed "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "friends";
 
     public static final List<Prefix> PREFIXES = List.of(PREFIX_ROOM, PREFIX_BOOKING_PERIOD, PREFIX_NAME, PREFIX_PHONE,
-            PREFIX_EMAIL, PREFIX_REMARK, PREFIX_TAG);
+            PREFIX_EMAIL, PREFIX_REMARK);
     public static final List<String> EXAMPLES = List.of("1", "2023-01-01 08:00 to 2023-01-02 12:00", "John Doe",
             "98765432", "johnd@example.com", "Requested extra Queen's sized bed", "friends");
```

Comment:
Just to check, the `PREFIXES` and `EXAMPLES` align with the change right?

Reply:
Good question, yes, the prefix no longer fetches that portion. I'll make another commit with that portion modified for clarity.